.changeset/cool-dingos-repair.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/theme': minor
+---
+
+Updates light theme's primary foreground and `running` status indicator colours to meet WCAG
```

Comment:
what do you think about adding the before and after colors to the changeset? just to make it easier for someone bumping their deps to know what impact it will have to their app?

Reply:
i can surely add that.